### ISSUE TITLE
Build a second frontend on the reusable Core

### ISSUE THREAD

**Issue by Extender92** [Extender92/Monopoly#6]
## Problem/Goal

Prove before clean publication that Core supports a second user interface without Console coupling, product-shaped assumptions, or the legacy project identity.

## Required behavior

- Record a documented technology and hosting decision before implementation.
- Deliver one thin vertical slice that can create or load a match, read state, submit a command or pending-decision response, and render resulting state and notifications.
- Use only the approved neutral identity and the original Demo profile.
- Consume public application/Core contracts without referencing Console types or mutable authoritative state.
- Support match-scoped notifications and resumable decisions across separate frontend calls.
- Report validation, command, decision, and persistence failures through typed results.
- Complete and verify the slice in the legacy fork before #58.
- Do not deploy, advertise, monetize, package, or publicly host the slice from the legacy repository.

## Scope

- Technology decision record.
- Minimal frontend composition, state projection, command/decision submission, and result rendering.
- One end-to-end automated or documented smoke path using the Demo profile.
- Verification that the code can be included in the clean export.

## Out of scope

- Full feature or visual parity with Console.
- Accounts, matchmaking, network multiplayer, payments, advertising, or monetization.
- Third-party presentation profiles or compatibility content.
- Deployment from the legacy repository.
- Clean-root publication and legacy-repository deletion.

## Acceptance criteria

- [ ] #4 and #37 provide the generic profile contract and original Demo profile.
- [ ] #49 supports resumable phases and stable pending decisions.
- [ ] #50 provides match-scoped disposable notifications.
- [ ] #51 prevents direct frontend mutation of authoritative state.
- [ ] #52 round-trips the complete match state used by the slice.
- [ ] #54 passes before frontend implementation is considered complete.
- [ ] #56 provides the approved neutral identity.
- [ ] A documented technology decision explains the frontend and hosting boundary.
- [ ] The slice creates or loads a Demo-profile match, reads state, submits a command or decision, and renders the result.
- [ ] Two simultaneous matches remain isolated.
- [ ] Automated tests cover the application boundary without real network calls or interactive input.
- [ ] No deployment, installer, package publication, advertisement, or monetization occurs in the legacy repository.
- [ ] The slice is included in #58 clean-export verification.
- [ ] Full Release build and tests pass without new warnings.

## Dependencies

- Requires #4, #37, #49 through #52, #54, and #56.
- Completes before #58; it is no longer blocked by repository migration or the publication epic.
- Public deployment, if later desired, is planned from a clean slate in the new repository.

## Related issues

- Final technical child before #58 and #59.
- Related to #32, #33, and #53.

**Comment by Extender92** [comment]
Deferred during neutral-Core backlog sanitation. This issue has not been implemented. It will not be copied to the clean repository and is not a cutover prerequisite. Reconsider the behavior only as a new, independently scoped neutral issue after cutover, based on the clean architecture and current product goals.
